### PR TITLE
Contributing.md highlighting and fixes

### PR DESCRIPTION
**Pull request by Gama11**
There were some duplicate closing brackets. Also, the indent example used a mix of spaces of tabs, which I don't think was intentional?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,26 +10,26 @@ If you want to add your plugin, theme or extension to AppMan, you need to do a p
 
 ##### Use spaces instead of tabs with a width of 4
 
-```
+```c#
 public bool IsMember()
```
Comment (Gama11) [inline CONTRIBUTING.md]: This should probably just be `return this.value <= 2;`. ;)

Comment (SlavaRa) [inline CONTRIBUTING.md]: + ;)

**Comment by Meychi** [comment]
Its should show as 4 spaces based on the page settings instead of what it actually contains. :)

**Comment by wise0704** [comment]
adskfjsa I literally was just about to make a pull request doing the same thing lol

**Pull request merged**